From 903ea980cf4568d5d8ad5abd3e02a8e5784e4a6e Mon Sep 17 00:00:00 2001
From: Shawn Lin <shawn.lin@rock-chips.com>
Date: Mon, 16 Jan 2017 17:36:27 +0800
Subject: [PATCH] arm64: dts: rockchip: replace clkreqn with cpm GPIO for
 rk3399-tve1205g

We should convert to use cpm mode instead of L1 substate.
Fix it anyway.

Change-Id: I5d997d53b2151ba9b1d29bd07272894a377c2eda
Signed-off-by: Shawn Lin <shawn.lin@rock-chips.com>
---
 arch/arm64/boot/dts/rockchip/rk3399-tve1205g.dts | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm64/boot/dts/rockchip/rk3399-tve1205g.dts b/arch/arm64/boot/dts/rockchip/rk3399-tve1205g.dts
index 61174f8f2dc0..49115e11cede 100644
--- a/arch/arm64/boot/dts/rockchip/rk3399-tve1205g.dts
+++ b/arch/arm64/boot/dts/rockchip/rk3399-tve1205g.dts
@@ -805,7 +805,7 @@
 	ep-gpios = <&gpio3 7 GPIO_ACTIVE_HIGH>;
 	num-lanes = <4>;
 	pinctrl-names = "default";
-	pinctrl-0 = <&pcie_clkreqn>;
+	pinctrl-0 = <&pcie_clkreqn_cpm>;
 	status = "okay";
 };
 
-- 
2.35.3

